packages/wcs-api/wcs-api.2017-05-26.01/opam

```diff
@@ -23,6 +23,7 @@ depends: [
   "wcs-lib" {= "2017-05-26.01"}
   "lwt_ssl"
   "cohttp-lwt-unix" {>= "0.20.0" & < "1.0"}
+  "jbuilder" {<="1.0+beta20.2"}
```

Comment:
```suggestion
  "jbuilder"
```
I'm not sure why add such a constraint. wcs seems to be able to build with any version of jbuilder (including transition): http://check.ocamllabs.io/log/1576115449-7b7ade392f48458c5399b9d5420146ef1b8605d2/4.02/good/wcs-lib.2017-05-26.02